spring-cloud-stream/src/test/java/org/springframework/cloud/stream/binder/BinderAwareChannelResolverTests.java

```diff
@@ -125,25 +125,24 @@ public void resolveNonRegisteredChannel() {
 	}
 
 	@Test
-	@SuppressWarnings("rawtypes")
+	@SuppressWarnings({"rawtypes", "unchecked"})
 	public void propertyPassthrough() {
 		ChannelBindingServiceProperties bindingServiceProperties = new ChannelBindingServiceProperties();
 		DynamicDestinationsBindable dynamicDestinationsBindable = new DynamicDestinationsBindable();
-		Map<String, BindingProperties> bindings = new HashMap<String, BindingProperties>();
-		BindingProperties bindingProperties = new BindingProperties();
-		bindingProperties.setContentType("text/plain");
```

Comment:
Although we have a separate issue about BindingProperties not being properly applied to channels.